backend/Tasque.BLL/Services/SprintService.cs

```diff
@@ -31,9 +31,20 @@ public async Task<IEnumerable<SprintDto>> GetProjectSprints(int projectId)
 
             return _mapper.Map<IEnumerable<SprintDto>>(sprints);
         }
+
```

Comment:
In this service using of FluentValidation namespace twice

Reply:
fixed this